pkg/cmd/pipelinerun/describe.go

```diff
@@ -97,6 +97,11 @@ func printPipelineRunStatus(w *tabwriter.Writer, pr *v1alpha1.PipelineRun, p cli
 	fmt.Fprintf(w, statusBody, formatted.Age(*pr.Status.StartTime, p.Time()),
 		formatted.Duration(pr.Status.StartTime, pr.Status.CompletionTime),
 		formatted.Condition(pr.Status.Conditions[0]))
+
+	if (formatted.IsFailed(pr.Status.Conditions[0])) {
```

Comment:
could you please move the function to this package itself; may be calling it `hasFailed` ?